### PR TITLE
console: show only compatible postgres functions in computed fields section (close #5155)

### LINKED ISSUE #5155: console: filter incompatible postgres functions from computed fields drop down
Only functions which have an argument which is of row type of the selected table are valid.

Menu_400

### PR DESCRIPTION
**Pull request by GrizliK1988**
Show only compatible postgres functions in computed fields section

### Description

When computed field is being created, user will see only  functions that have table row as one of arguments

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Console

### Related Issues

#5155 

### Solution and Design

At the beginning I planned to adjust SQL query to fetch only needed functions like
```
SELECT * FROM hdb_catalog.hdb_function_agg WHERE "input_arg_types"::jsonb @> '[{"schema":"table_schema","name":"table_name"}]'::jsonb;
```

Unfortunately backend doesn't accept json fields in where statements and I adjusted filtering on js side, which shouldn't be a problem, until user will have thousands of functions. 

### Steps to test and verify

There is a new test case in Cypress to verify functionality.

### CODE REVIEW

**Review by shaonkabir8 — APPROVED** [review]

**Review by beerose — CHANGES_REQUESTED** [review]
Thanks for your PR!

Filtering out functions that don't accept table row argument works great!

However, we could go one step forward and handle a few more requirements mentioned here: https://hasura.io/docs/1.0/graphql/core/schema/computed-fields.html#supported-sql-functions.

Specifically, we could:
* Check for the `function_type` and accept only STABLE or IMMUTABLE
* Check for the input arg types (we need 'b' — base or 'c' — composite)
Screenshot 2020-10-16 at 19 16 48

File: console/cypress/integration/data/modify/test.ts [hunk 1]
```diff
@@ -40,7 +42,9 @@ export const runModifyTableTests = () => {
   describe('Modify Table', () => {
     it('Creating a table', passMTCreateTable);
     it('Moving to the table', passMTMoveToTable);
+    it('Creating functions', passMTCreateFunctions);
     it('Modify table button opens the correct route', passMTCheckRoute);
+    it('See only functions for current table', passMTFunctionList);
```
[the remark this answers is not in the document]
Comment (GrizliK1988) [reply console/cypress/integration/data/modify/test.ts:47]: Hi @beerose,

Thank you for your feedback! Described changes look good to me and I'll apply them in upcoming days.

[the remark this answers is not in the document]
Comment (GrizliK1988) [reply console/cypress/integration/data/modify/test.ts]: Hi!

Code is updated. VOLATILE functions or functions with unsupported arguments are now filtered out.

**Review by beerose — APPROVED** [review]
Thank you again for your PR and for taking time to add tests! 🎉

Approving console changes

**Review by beerose — APPROVED** [review]
Console

**Pull request closed without merging**